src/utils/__tests__/concat_map_latest.test.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2015 CANAL+ Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { expect } from "chai";
+
+import {
+  from as observableFrom,
+  Observable,
+  of as observableOf,
+  timer,
+} from "rxjs";
+import {
+  concatMap,
+  delay,
+  mapTo,
+} from "rxjs/operators";
+import concatMapLatest from "../concat_map_latest";
+
+describe("utils - concatMapLatest", () => {
+  it("should output all inner values", (done) => {
```

Comment:
that's a weird test case text.

This tests more something like
`should output all inner values if something something`